.jenkins/continuous.groovy

```diff
@@ -351,12 +351,14 @@ pipeline {
                         sh 'ccache --zero-stats'
                         sh 'rm -rf build && mkdir -p build'
                         dir('build') {
+                            // FIXME_CXX17: remove CMAKE_CXX_STANDARD once we require C++17
                             sh '''
                                 cmake \
                                     -D CMAKE_INSTALL_PREFIX=$ARBORX_DIR \
                                     -D CMAKE_BUILD_TYPE=Release \
                                     -D CMAKE_CXX_COMPILER=clang++ \
                                     -D CMAKE_CXX_EXTENSIONS=OFF \
+                                    -D CMAKE_CXX_STANDARD=17 \
```

Comment:
The SYCL backend requires C++17.  You don't need the FIXME

Reply:
Then I technically don't need this line either, as the standard will come from Kokkos anyway?

Reply:
No you need it